src/App.tsx

```diff
@@ -1,14 +1,70 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { Options } from './types/Options';
+import { getTodos } from './api';
+
+function getFilteredTodos(todos: Todo[], filter: Filter): Todo[] {
+  let filteredTodos = [...todos];
+
+  filteredTodos = filteredTodos.filter(todo => {
```

Comment:
unnecessary assignment
```suggestion
  filteredTodos.filter(todo => {
```